README.md

```diff
@@ -23,9 +23,13 @@ In case of certificates and token you need to setup the following directory stru
       * prod_key.pem - Production APNS app certificate's private key (has to be unencrypted)
       * dev_cert.pem - Development APNS app certificate
       * dev_key.pem - Development APNS app certificate's private key (has to be unencrypted)
+      * token.p8 - `APNS` authentication token
     * fcm/
       * token.json - `FCM` service account JSON file
 
+In case you want to use `APNS` token authentication you need to provide token and set `key_id` and `team_id` environmental variables. On how to obtain token and `key_id` read:
```

Comment:
If you want to use the `APNS` token authentication, you need to provide a token and set the `key_id` and `team_id` environmental variables. To see how to obtain token and `key_id` read: